crates/uv/tests/common/mod.rs

```diff
@@ -128,33 +175,35 @@ impl TestContext {
         .stdout(version);
     }
 
-    /// Generate an escaped regex pattern for the given path.
+    /// Generate various escaped regex patterns for the given path.
     fn path_patterns(path: impl AsRef<Path>) -> Vec<String> {
-        vec![
-            format!(
-                // Trim the trailing separator for cross-platform directories filters
-                r"{}\\?/?",
-                regex::escape(
-                    &path
-                        .as_ref()
-                        .canonicalize()
-                        .expect("Failed to create canonical path")
-                        // Normalize the path to match display and remove UNC prefixes on Windows
-                        .simplified()
-                        .display()
-                        .to_string(),
-                )
-                // Make seprators platform agnostic because on Windows we will display
+        let mut patterns = Vec::new();
+
+        // We can only canonicalize paths that exist already
+        if path.as_ref().exists() {
+            patterns.push(Self::path_pattern(
+                path.as_ref()
+                    .canonicalize()
+                    .expect("Failed to create canonical path"),
+            ));
+        }
```

Comment:
This needs to be conditional because sometimes the site packages directory does not exist.